Doppler.PushContact/Controllers/DomainController.cs

```diff
@@ -0,0 +1,29 @@
+using Doppler.PushContact.DopplerSecurity;
+using Doppler.PushContact.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace Doppler.PushContact.Controllers
+{
+    [Authorize(Policies.ONLY_SUPERUSER)]
+    [ApiController]
+    public class DomainController : ControllerBase
+    {
+        [HttpPut]
+        [Route("domains/{name}")]
+        public Task<IActionResult> Upsert([FromRoute] string name, [FromBody] bool isPushFeatureEnabled)
+        {
+            throw new NotImplementedException();
+        }
+
+        [AllowAnonymous]
+        [HttpGet]
+        [Route("domains/{name}")]
+        public Task<ActionResult<Domain>> Get(string name)
```

Comment:
I am not sure how the Domain model will evolve.

If the future new fields of the Domain model are going to be useful for the external websites, this approach is the best one. 

If the future new fields of the Domain model are going to be for internal usage only, maybe this is better:

 ```suggestion
        [AllowAnonymous]
        [HttpGet]
        [Route("domains/{name}/isPushFeatureEnabled")]
        public Task<ActionResult<boolean>> Get(string name)
```

Reply:
At the moment, the external websites just will use `isPushFeatureEnabled` field. I think that future new fields like _createdDate_ or _lastModifiedDate_ will be not necessary for consumers. 

_domains/{name}/isPushFeatureEnabled_ is all we need. Thanks

Reply:
> domains/{name}/isPushFeatureEnabled is all we need

Done! Related diagram updated too